flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/DefaultScheduler.java

```diff
@@ -401,12 +401,17 @@ private static void propagateIfNonNull(final Throwable throwable) {
 					.registerProducedPartitions(logicalSlot.getTaskManagerLocation(), sendScheduleOrUpdateConsumerMessage);
 				executionVertex.tryAssignResource(logicalSlot);
 			} else {
-				handleTaskFailure(executionVertexId, maybeWrapWithNoResourceAvailableException(throwable));
+				onTaskDeploymentFailure(executionVertexId, maybeWrapWithNoResourceAvailableException(throwable));
 			}
 			return null;
 		};
 	}
 
+	private void onTaskDeploymentFailure(final ExecutionVertexID executionVertexId, final Throwable error) {
+		log.info("An error is detected when trying to schedule or deploy task {}.", executionVertexId, error);
```

Comment:
Could be a bit more concise: `"Error while scheduling or deploying task {}"`. Let me know what you think.

Reply:
Ok. done.